src/main/java/edu/harvard/h2ms/domain/core/Notification.java

```diff
@@ -71,6 +71,9 @@
   @Column(name = "notification_body")
   private String notificationBody;
 
+  @ElementCollection(fetch = FetchType.EAGER)
+  Map<String, String> notificationParameters;
```

Comment:
Seems strange that `notification` is prefixed to fields in this class. Since being in the Notification class implies properties belong to the notification.